types/ember/index.d.ts

```diff
@@ -3246,9 +3221,8 @@ declare module 'ember' {
          * Forces the passed object to be part of an array. If the object is already
          * an array, it will return the object. Otherwise, it will add the object to
          * an array. If obj is `null` or `undefined`, it will return an empty array.
-         * @private
          */
-        function makeArray<T>(obj?: T[] | T | null | undefined): T[];
+        function makeArray<T>(obj?: T[] | T | null): T[];
```

Comment:
I assume `undefined` is removed here as redundant given the `obj?`?

Reply:
Yep, you can pass `undefined` already.